cdap-docs/developer-manual/source/cloud-runtimes/provisioners/remote-hadoop.rst

```diff
@@ -8,5 +8,28 @@
 Remote Hadoop
 =============
 
-Describe what the remote hadoop provisioner is for and each config setting
+The Remote Hadoop provisioner is used to run jobs on a pre-existing Hadoop
```

Comment:
The Remote Hadoop provisioner is used to run jobs on a pre-existing Hadoop cluster. This provisioner does not create or delete a cluster; it simply connects to one. We assume that the provided cluster is running compatible versions of all the software that CDAP requires. The cluster must not use Kerberos authentication and must be accessible via SSH and HTTPS (ports 22 and 443).

The configuration for the Remote Hadoop provisioner is the same information required to SSH to an edge node of the cluster.